### PR TITLE
Fix diff code block highlight color, change the format of yaml diff code block from raw diff to git diff.

### PR DESCRIPTION
**Pull request by IsQiao**
Fix wrong highlight in diff code block

#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

Change the format of yaml diff code block from raw diff to git diff.

#### Which issue(s) this PR fixes:

None

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by Huang-Wei** [comment]
> Fix wrong highlight in diff code block

Maybe reword it as "Change the format of yaml diff code block from raw diff to git diff"

**Comment by IsQiao** [comment]
> > Fix wrong highlight in diff code block
> 
> Maybe reword it as "Change the format of yaml diff code block from raw diff to git diff"

Updated

**Comment by IsQiao** [comment]
Is there anything else I need to do? @Huang-Wei

**Review by Huang-Wei — COMMENTED** [review]
/lgtm
/approve

Thanks.
